apps/modernization-ui/src/pages/advancedSearch/components/InvestigationResults.spec.tsx

```diff
@@ -76,7 +76,6 @@ describe('InvestigationResults component tests', () => {
         expect(getByText('2019 Novel Coronavirus')).toBeInTheDocument();
         expect(getByText('CAS10003001GA99')).toBeInTheDocument();
         expect(getByText('1/1/1990')).toBeInTheDocument();
-        expect(getByText(`(${age} years)`)).toBeInTheDocument();
```

Comment:
Issue: if this is failing, it should be fixed not removed

Reply:
inserted back. it works